docs/formats/README.md

```diff
@@ -1,8 +1,9 @@
 # Zed Formats
 
-This directory contains specifications for the Zed family of data formats,
-providing a unified approach to row, columnar, and human-readable formats.
-The Zed data model underlying the formats
+This directory contains specifications for the Zed family of
+["superstructured"](#zed-data-model.md-2-zed-a-super-structured-design-pattern)
```

Comment:
I think we should style this term as "super-structured" both for agreement with "semi-structured" and to avoid any confusion with existing usage of "superstructure" as a noun.

Reply:
Yeah I struggled with this a bit since it is a word, but your idea is better.